### PR TITLE
Arm PR to master branch

### PR DESCRIPTION
**Pull request by JohnBurdick1**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/arm/ArmBase.java [hunk 1]
```diff
@@ -6,57 +6,59 @@
...
-
         baseMotor.configFactoryDefault();
         baseMotor.enableVoltageCompensation(true);
-        SupplyCurrentLimitConfiguration limit = new SupplyCurrentLimitConfiguration(true, 40, 80, 0.5);
+        SupplyCurrentLimitConfiguration limit = new SupplyCurrentLimitConfiguration(true, 20, 25, 0.5);
         baseMotor.configSupplyCurrentLimit(limit);
         baseMotor.config_kP(0, kP_BASE);
         baseMotor.config_kI(0, kI_BASE);
         baseMotor.config_kD(0, kD_BASE);
+        baseMotor.setSelectedSensorPosition(angleToTick(90));
```
Comment (mmarchetti) [inline src/main/java/frc/robot/subsystems/arm/ArmBase.java:45]: Note for future reference: until we have a cancoder, we are powering up the arm at 90 degrees and hardcoding that position.

File: src/main/java/frc/robot/subsystems/arm/ArmWrist.java [hunk 2]
```diff
@@ -1,63 +1,66 @@
...
+        SupplyCurrentLimitConfiguration limit = new SupplyCurrentLimitConfiguration(true, 40, 60, 0.5);
         wristMotor.configSupplyCurrentLimit(limit);
-        feedforward = new ArmFeedforward(kS_WRIST, kG_WRIST, kV_WRIST);
-        wristPotentiometer = new AnalogPotentiometer(
-                WRIST_POTENTIOMETER_PORTNUM, ARM_WRIST_POTENTIOMETER_MULT, ARM_WRIST_POTENTIOMETER_ADD);
-        // offset 0 is a placeholder, due to the fact we have no means of
-        // determining actual degree offset right now
+        wristMotor.config_kP(0, kP_ELBOW);
+        wristMotor.config_kI(0, kI_ELBOW);
+        wristMotor.config_kD(0, kD_ELBOW);
```
Comment (mmarchetti) [inline src/main/java/frc/robot/subsystems/arm/ArmWrist.java:42]: These should be the wrist PID constants.

File: src/main/java/frc/robot/subsystems/arm/ArmWrist.java [hunk 3]
```diff
@@ -1,63 +1,66 @@
...
+        feedforward = new ArmFeedforward(kS_ELBOW, kG_ELBOW, kV_ELBOW, kA_ELBOW);
```
Comment (mmarchetti) [inline src/main/java/frc/robot/subsystems/arm/ArmWrist.java:43]: these too

File: src/main/java/frc/robot/subsystems/arm/ArmWrist.java [hunk 4]
```diff
@@ -1,63 +1,66 @@
...
+    
     public double getAngle() {
-        return wristPotentiometer.get();
+        return wristMotor.getSelectedSensorPosition();
     }
 
     public static double angleToTick(double angle) {
         double revolutionsOfArm = angle / 360.0;
-        double motorRevolutions = revolutionsOfArm * ARM_WRIST_GEAR_RATIO;
+        double motorRevolutions = revolutionsOfArm * INV_GEAR_RATIO_BASE;
```
Comment (mmarchetti) [inline src/main/java/frc/robot/subsystems/arm/ArmWrist.java:63]: wrist ratio

**Review by MathNerd28 — APPROVED** [review]

**Pull request merged**